frontend/package.json

```diff
@@ -6,6 +6,7 @@
     "axios": "^0.19.0",
     "bootstrap": "^4.4.1",
     "bootswatch": "^4.3.1",
+    "decko": "^1.2.0",
```

Comment:
decko is a cool library that makes binding to local functions easier. You'll see me use this as a decorator in code.